runtime/near-vm/engine/src/universal/code_memory.rs

```diff
@@ -212,12 +212,13 @@ impl Drop for CodeMemory {
                     );
                 }
             }
-            drop(source_pool.push(Self {
+            let mut guard = source_pool.lock().unwrap_or_else(|e| e.into_inner());
```

Comment:
I’m not sure I feel comfortable ignoring the poison here. Considering we’re currently on panic=abort anyway, AFAICT this cannot happen. So maybe just unwrap the lock?

Reply:
I'm just following a broad community consensus from a couple years ago that poisoning isn't actually doing much useful and just defaulting to ignoring the poison unless clearly necessary to handle it for correct operation. There are good rules of thumb etc.

But anyway since panic=abort, I don't care much either way.